docs/ssm-deployment-guide.md

```diff
@@ -400,7 +402,7 @@ the configuration takes effect. You can try TestDFSIO for example,
 
 	  `hadoop jar ./share/hadoop/mapreduce/hadoop-mapreduce-client-jobclient-2.6.0-cdh5.10.1-tests.jar TestDFSIO –read`
 
-   You may want to replace the jar with the version used in your cluster. After the read data operation, if all the data files are listed on SSM web UI page "hot files" table, then the integration works very well. 
+   You may want to replace the jar with the version used in your cluster. After the read data operation, if all the data files are listed on SSM web UI page "hot files" table, the integration works very well.
```

Comment:
After the read operation, if all the data files are listed in "hot files" table on SSM web UI page, it proves that the integration works very well.